io.github.giantpinkrobots.varia.json

```
@@ -0,0 +1,60 @@
+{
+    "app-id" : "io.github.giantpinkrobots.varia",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "varia",
+    "//": "Justifications for the permissions: This program utilizes aria2c, and aria2c requires the networking permission and the xdg-download directory in order to download files.",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=xdg-download"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	"./python3-aria2p.json",
+	{
+	   "name" : "aria2",
+	   "buildsystem" : "simple",
+	   "build-commands" : [
+	   	"mkdir /app/aria2/",
+		"./configure --without-libxml2 --without-libexpat --without-sqlite3 --without-appletls --without-gnutls --without-libgmp --without-libnettle --without-libgcrypt --without-libssh2 --without-cppunit --without-libz --without-libcunit --without-libssl --without-libuv --without-libcares --without-libaria2 --disable-nls --with-openssl --disable-bittorrent --disable-ftp",
+	   	"make",
+	   	"install -Dm755 src/aria2c /app/aria2/"
+	   ],
```

```suggestion
	   "buildsystem" : "autotools",
	   "config-opt": [
		"--without-libxml2",
		"--without-libexpat",
		"--without-sqlite3",
		"--without-appletls",
		"--without-gnutls",
		"--without-libgmp",
		"--without-libnettle",
		"--without-libgcrypt",
		"--without-libssh2",
		"--without-cppunit",
		"--without-libz",
		"--without-libcunit",
		"--without-libssl",
		"--without-libuv",
		"--without-libcares,
		"--without-libaria2",
		"--disable-nls",
		"--with-openssl",
		"--disable-bittorrent",
		"--disable-ftp"
	   ],
	   "no-make-install": true,
	   "post-install": [
	       "install -Dm755 src/aria2c /app/aria2/"
	   ],
```

this assuming these configure option are actually necessary and the install rule doesn't do what you want. I DID NOT CHECK.

Also indentation.

---

The config options are there because there are some features that are not needed (for example features relating to Bittorrent, my program doesn't support that) and thus I wanted to optimize it for faster compile times and smaller footprint.